src/os/shared.rs

```diff
@@ -182,7 +191,7 @@ pub fn get_input(
 
     let network_frames = network_interfaces
         .iter()
-        .filter(|iface| iface.is_up() && !iface.ips.is_empty())
+        .filter(|iface| !iface.ips.is_empty())
```

Comment:
This is a regression, but I had to do that as on Windows libpnet does not read interface flags which causes ```iface.is_up()``` to fail.

Reply:
Yup, its a TODO in libpnet, I guess conditional is a way to go for now.
```
         all_ifaces.push(NetworkInterface {
                name: name_str,
                index: (*cursor).Index,
                mac: Some(mac),
                ips: ips,
                // flags: (*cursor).Type, // FIXME [windows]
                flags: 0,
            });
```